### ISSUE TITLE
idea: Include changes to outputs in pull request comments

### ISSUE THREAD

**Issue by ricardofalc** [borchero/terraform-plan-comment#24]
Hi,

I've noticed that changes to outputs from the terraform plan are not being reflected in the pull request comments generated by this action.

Could you clarify if this is a design choice or a potential enhancement? It would be really helpful to have these output changes listed along with other modifications in the plan for better visibility during code reviews.

Thanks for the great work on this action! I’m looking forward to your feedback.

Comment in PR indicates there are no changes:
![image](https://github.com/user-attachments/assets/dcfa1a75-11a0-4afc-aa3e-588448fb5fca)

Terraform plan in workflow indicates the following:
![image](https://github.com/user-attachments/assets/920f6d19-9875-4190-8810-e0e79c0b6aa1)

**Comment by borchero** [comment]
Good catch, I think that would be a great addition to this action! Unfortunately, it is currently unsupported.

I'd be happy to review a PR for this, it would take me a some time until I'd find the time to see through the implementation myself :eyes:

**Comment by fumantsu** [comment]
not stale.

**Comment by fumantsu** [comment]
not stale
